org.eclipse.jdt.ls.core/plugin.xml

```diff
@@ -97,6 +97,12 @@
             <command
                   id="java.project.resolveStackTraceLocation">
             </command>
+            <command
+                  id="java.action.openTypeHierarchy">
+            </command>
+            <command
+                  id="java.action.resolveTypeHierarchy">
+            </command>
```

Comment:
By default, the delegate command will be exposed as a command in client side. But these two commands are already in use by your client PR, where you invoke `commands.registerCommand(Commands.OPEN_TYPE_HIERARCHY, ...)` to register the same commands in client. That's probably why Roland saw "command 'java.action.resolveTypeHierarchy' not found" error.

You should use a different name such as "java.navigate.openTypeHierarchy", "java.navigate.resolveTypeHierarchy".

Reply:
Sure. I'll rename these commands.